eng/native/configureplatform.cmake

```diff
@@ -434,7 +434,7 @@ if(CLR_CMAKE_TARGET_OS STREQUAL haiku)
     set(CLR_CMAKE_TARGET_HAIKU 1)
 endif(CLR_CMAKE_TARGET_OS STREQUAL haiku)
 
-if(CLR_CMAKE_TARGET_OS STREQUAL emscripten)
+if(CLR_CMAKE_TARGET_OS STREQUAL emscripten OR CLR_CMAKE_TARGET_OS STREQUAL browser)
```

Comment:
why is this needed?

Reply:
When I run `build.cmd -os browser`, the `CLR_CMAKE_TARGET_OS` is set to `browser`. Where is `CLR_CMAKE_TARGET_OS` set to `emscripten`? Maybe we should make it work similar on coreclr then?